functions/delete-all/1.0/index.js

```diff
@@ -0,0 +1,70 @@
+const take = 200;
+
+const deleteAll = async ({ model }) => {
+  try {
+    const { name: modelName } = model;
+    const { totalCount } = await getAll(modelName, 'totalCount');
+    const maxRequests = Math.ceil(totalCount / take);
+
+    // Loop through batches
+    for (let index = 0; index < maxRequests; index += 1) {
+      await deleteBatch(modelName);
+    }
+    return {
+      result: `All records from ${modelName} have been deleted (${totalCount})`,
+    };
+  } catch (error) {
+    if (error instanceof TypeError) {
+      throw new Error(
+        `Something went wrong while deleting all records from ${modelName}`,
+      );
+    } else {
+      throw error;
+    }
+  }
+};
+
+const deleteBatch = async (modelName) => {
```

Comment:
Can you move the `deleteBatch` and the `getAll` functions above the `deleteAll` function